src/azul/queues.py

```diff
@@ -220,9 +220,12 @@ def wait_to_stabilize(self) -> int:
         Wait for the work queues to reach a steady state.
         """
         sleep_time = 10
+        # The timeout is determined by the max message visibility timeout set by
+        # a retry lambda, in this case the contribution lambda is 15 min.
```

Comment:
Still brittle. And don't explain the how, explain the why. Consider asking team a mate for help with the text.